documentation/docs/steps/cfManifestSubstituteVariables.md

```diff
@@ -0,0 +1,55 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+This step is only relevant if both a `manifest.yml` and a corresponding variables Yaml file are found at the specified paths in the current source tree.
+The step will activate itself in this case, and tries to replace any variable references found in `manifest.yml` with the values found in the variables file.
+
+**Note:** It is possible to use one variables file for more than one `manifest.yml`.
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Side effects
+
+Unless configured otherwise, this step will *replace* the input `manifest.yml` with a version that has all variable references replaced. This alters the source tree in your Jenkins workspace.
```

Comment:
It is intended, but it is a side effect. Since the section is just called side effects and not "unintended side effects", I thought it would fit. Also, in case Jenkins keeps printing errors that I cannot checkout the latest sources, because something has changed in the current source tree, it might help to know why it has changed. 
I can remove it, of course.